Jenkinsfile_CNP

```diff
@@ -7,13 +7,15 @@ import uk.gov.hmcts.contino.HealthChecker
 
 properties([
         parameters([
-                string(name: 'PRODUCT_NAME', defaultValue: 'ccd', description: ''),
+                string(name: 'PRODUCT_NAME', defaultValue: 'probate', description: 'probate|ccd|..'),
```

Comment:
Let's use a dropdown for this please.  And then get the database from a lookup. too easy to make a mistake.